### DISCUSSION TITLE
Why are the images rotated after they are compressed?

### DISCUSSION THREAD

**Discussion by alexanderaxentioi** [joshembling/image-optimizer#8] [Q&A] [answered]
Why are the images rotated after they are compressed? I want to use the compressor but it seems like it's rotating all of my images  after the compression, is there a way to not rotate them?

**Comment by joshembling** [comment, marked as answer]
Hi @alexanderaxentioi, I've never seen this happen. Are you able to send me a demo repository where this is happening for you?
